### PR TITLE
Improved testing options

### PR DESCRIPTION
**Pull request by tunetheweb**
Improved version of #286 

Changes:
- Allows you to run: `npm run test:e2e -- --browser=chrome --metric=TTFB`.
- Adds a precommit test for any stray `it.only`'s or `browser.debug`'s left around.
- Adds instructions to contributing file.
- Moves Code of Conduct and Contributing files to more commonly known locations to make them easier to find.

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -2,3 +2,11 @@
 . "$(dirname -- "$0")/_/husky.sh"
 
 npm exec lint-staged
+
+echo "Checking for leftover it.only's..."
```
Comment (philipwalton) [inline .husky/pre-commit:6]: Can you also check for `describe.only`?

**Comment by tunetheweb** [comment]
Closing in favour of #460

File: .husky/pre-commit (hunk 1 above)
Comment (tunetheweb) [reply .husky/pre-commit:6]: Done in #460

File: docs/contributing.md [hunk 2]
```diff
@@ -22,6 +22,20 @@ use GitHub pull requests for this purpose. Consult
...
+
+To test an individual browser or metric, run the following in separate terminals:
+
+- `npm run watch`
+- `npm run test:server`
+- `npm run test:e2e -- --browser=chrome --metric=TTFB`
+
+To run an individual test, change `it('test name')` to `it.only('test name')`.
+
+You can also add `await browser.debug()` lines to the individual test files to pause execution, and press `CTRL+C` in the command line to continue the tests.
```
[the remark this answers is not in the document]
Comment (tunetheweb) [reply docs/contributing.md:37]: Done in #460

**Pull request closed without merging**